wp-includes/class-wp-https-ui.php

```diff
@@ -157,6 +149,17 @@ public function render_https_settings() {
 			) );
 			?>
 		</p>
+
+		<?php if ( is_wp_error( $https_support ) ) : ?>
+			<?php foreach ( $https_support->get_error_messages() as $error_message ) : ?>
+				<div class="notice notice-error inline">
+					<p>
+						<?php echo esc_html( $error_message ); ?>
```

Comment:
It's great how this displays the error message(s):


![error-message-j](https://user-images.githubusercontent.com/4063887/45375486-b92bcc80-b5ba-11e8-9485-9467f0c14812.jpg)